src/js/backup_restore.js

```diff
@@ -847,8 +863,13 @@ function configuration_restore(callback) {
                     FC.GPS_CONFIG = configuration.GPS_CONFIG;
                     FC.RSSI_CONFIG = configuration.RSSI_CONFIG;
                     FC.BOARD_ALIGNMENT_CONFIG = configuration.BOARD_ALIGNMENT_CONFIG;
-                    FC.CONFIG.name = configuration.CRAFT_NAME;
-                    FC.CONFIG.displayName = configuration.DISPLAY_NAME;
+                    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
+                        FC.CONFIG.craftName = configuration.CRAFT_NAME;
+                        FC.CONFIG.pilotName = configuration.PILOT_NAME;
+                    } else {
+                        FC.CONFIG.name = configuration.CRAFT_NAME;
+                        FC.CONFIG.displayName = configuration.DISPLAY_NAME;
+                    }
```

Comment:
This one could similary be refactored to use `??` and defaulting to things. Again up for discussion.